spec/shoryuken/environment_loader_spec.rb

```diff
@@ -4,6 +4,36 @@
 RSpec.describe Shoryuken::EnvironmentLoader do
   subject { described_class.new({}) }
 
+  describe '#load' do
+    before do
+      Shoryuken.groups.clear
+      # See issue: https://stackoverflow.com/a/63699568 for stubbing AWS errors
+      allow(Shoryuken::Client)
+        .to receive(:queues)
+        .with('stubbed_queue')
+        .and_raise(Aws::SQS::Errors::NonExistentQueue.new(nil, nil))
```

Comment:
I guess looking at this, I can do something else similar to - https://stackoverflow.com/a/54314792. It could be better to not depend on an underlying API which is susceptible to change in the specs, otherwise if AWS SDK changes number of parameters, etc. to the `Aws::ServiceError` class, this test won't break. WDYT? @cjlarose

Reply:
I didn't know about `stub_const` but it looks like it might be worth trying here!